### PR TITLE
Allow disabling pagination press with a prop

### PR DESCRIPTION
**Pull request by jasperSpeicher**
#93

### CODE REVIEW

File: src/components/Pagination/Pagination.tsx [hunk 1]
```diff
@@ -32,6 +32,7 @@ export type PaginationProps = {
   paginationStyleItemActive?: ViewStyle;
   paginationStyleItemInactive?: ViewStyle;
   onPaginationSelectedIndex?: () => void;
+  disablePaginationPress?: boolean;
```
Comment (gusgard) [inline src/components/Pagination/Pagination.tsx:35]: Could you rename this to paginationIsDisabled

Comment (jasperSpeicher) [reply src/components/Pagination/Pagination.tsx:35]: I can, but I think `paginationIsDisabled` implies that there will be no pagination. 

What is it about this name that you don't like?

Would you consider `paginationTapDisabled` or `paginationInteractionsDisabled`?

Comment (gusgard) [reply src/components/Pagination/Pagination.tsx:35]: Both paginationTapDisabled or paginationInteractionsDisabled are good for me. I want to keep the idea of using a prefix, eg `paginationXXXX`

Comment (MaganAnkur) [reply src/components/Pagination/Pagination.tsx:35]: can we merge it?

Comment (martinmanzo) [reply src/components/Pagination/Pagination.tsx:35]: Hi @gusgard , would you consider merging this if I submit a PR with the name `paginationTapDisabled`?

Comment (jasperSpeicher) [reply src/components/Pagination/Pagination.tsx:35]: See https://github.com/gusgard/react-native-swiper-flatlist/pull/129

Comment (gusgard) [reply src/components/Pagination/Pagination.tsx:35]: Yes, I will merge it in a bit

Comment (gusgard) [reply src/components/Pagination/Pagination.tsx:35]: Released with https://github.com/gusgard/react-native-swiper-flatlist/releases/tag/3.0.16

**Pull request closed without merging**